lib/pbench/test/unit/server/test_datasets_metadata.py

```diff
@@ -76,6 +67,18 @@ def test_get2(self, client, server_config, provide_metadata):
             "user": {"contact": "me@example.com"},
         }
 
+    def test_get_bad_query(self, client, server_config, provide_metadata):
+        response = client.get(
+            f"{server_config.rest_uri}/datasets/metadata",
+            query_string={
+                "name": "drb",
+                "controller": "foobar",
+                "metadata": "dashboard.seen,server.deletion,dataset.access,user",
+            },
+        )
+        assert response.status_code == HTTPStatus.BAD_REQUEST
+        assert response.json == {"message": "Unknown URL query keys: controller"}
```

Comment:
When/where did `"controller"` get removed from the schema?

Reply:
To (mis)quote George Lucas, "Moderately long ago, in a galaxy not directly adjacent".

It went bye-bye when I re-wrote `datasets/list` to use SQL instead of Elastic.

Reply:
Should I be concerned about the fact that the tests didn't object...or...are these not the droids I'm looking for?  (I can go about my business...I'll just move along, now....)

Reply:
No, those droids went "thataway" ...

I was previously ignoring unrecognized query parameters. When I was writing the new common handler it occurred to me that it would be easy to misspell one, and it'd be silently ignored, so I added the error checking as in our `KEYWORD` schema type.

And of course immediately tests failed because of these obsolete (and previously ignored) `controller` params.